### PR TITLE
Incorporate Breadth into model evaluation

### PR DESCRIPTION
**Pull request by namn12**
This is a pretty simple merge, but I just want to get your opinion before I finish it. Should I just add a `switch_T` argument to the function argument of the function in Fig_Gen. If it's true its switch the lineage generation part of the functions, but all else stays the same.

### CODE REVIEW

**Comment by shak360** [comment]
@namn12 Did you wanna mark this ready for review?

File: lineage/figures/Breadth_Two_State_Lineage.py [hunk 1]
```diff
@@ -9,11 +9,29 @@ def Breadth_Two_State_Lineage(experimentTime, initCells, locBern, betaExp, switc
 
     LINEAGE = gpt(experimentTime, initCells, locBern, betaExp, switchT, bern2, betaExp2, FOM)
 
+    LINEAGE = remove_singleton_lineages(LINEAGE)
```
Comment (shak360) [inline lineage/figures/Breadth_Two_State_Lineage.py]:
```suggestion
    LINEAGE = remove_unfinished_cells(LINEAGE)
    LINEAGE = remove_singleton_lineages(LINEAGE)
```

File: lineage/figures/Breadth_Two_State_Lineage.py [hunk 2]
```diff
@@ -9,11 +9,29 @@ def Breadth_Two_State_Lineage(experimentTime, initCells, locBern, betaExp, switc
...
+        
     while not LINEAGE:  # determines if lineage is empty, so can regenerate a new one
         LINEAGE = gpt(experimentTime, initCells, locBern, betaExp, switchT, bern2, betaExp2, FOM)
+        LINEAGE = remove_singleton_lineages(LINEAGE)
```
Comment (shak360) [inline lineage/figures/Breadth_Two_State_Lineage.py]:
```suggestion
        LINEAGE = remove_unfinished_cells(LINEAGE)
        LINEAGE = remove_singleton_lineages(LINEAGE)
```

File: lineage/figures/Breadth_Two_State_Lineage.py [hunk 3]
```diff
@@ -9,11 +9,29 @@ def Breadth_Two_State_Lineage(experimentTime, initCells, locBern, betaExp, switc
...
+    for ii, cell in enumerate(X):
+        if cell.true_state == 0:
+            masterLineage.append(cell)
+        elif cell.true_state == 1:
+            sublineage2.append(cell)
+        else:
+            raise print('more than 2 true states error')
+            
+
+    if verbose:
```
Comment (shak360) [inline lineage/figures/Breadth_Two_State_Lineage.py]: Should do `logging` for this later

**Comment by namn12** [comment]
> @namn12 Did you wanna mark this ready for review?

I'm going to finish working on it tomorrow. and have it up for review in the afternoon/night

**Comment by namn12** [comment]
can we first merge #109 to master? And then merge master to this branch. Because i just want to make sure with all the figure formatting that is in #109, this PR will still function properly. This PR uses all the formatting the is updated in #109

File: lineage/figures/Breadth_Two_State_Lineage.py (hunk 3 above)
Comment (namn12) [reply lineage/figures/Breadth_Two_State_Lineage.py]: done

**Comment by aarmey** [comment]
Merge conflicts here.

**Comment by shak360** [comment]
image
@namn need a little help with this one

**Comment by shak360** [comment]
image
looks like the `MASbeta` might be a list instead of a scalar value, might just be an incorrect variable name thing

**Comment by shak360** [comment]
fixed lol... we forgot that `switchT` breadth lineages use lists for the "true" values, added some simple code that takes those lists back to single values

**Pull request merged**